deploy/README.md

```diff
@@ -16,6 +16,7 @@ This page has instructions for collecting Kubernetes logs, metrics, and events;
         - [Manual Source Creation and Setup](#manual-source-creation-and-setup)
             - [Create a hosted collector and an HTTP source](#create-a-hosted-collector-and-an-http-source)
             - [Deploy Fluentd](#deploy-fluentd)
+        - [Events](#events)
```

Comment:
we're missing the subsection `Configurable fields for events`.

To auto-generate the TOC I have been using the "Auto Markdown TOC" plugin for VS Code.